TurtleArt/tabasics.py

```diff
@@ -560,7 +564,18 @@ def _color_palette(self):
                             _('black'))
         for name in color_names:
             self._make_constant(palette, name, _(name), name)
-
+        
+        palette.add_block('RGB to color',
+                        style='number-style-var-3arg',
+                        label=[_('RGB to \ncolor'), _('Red'), _('Green'), _('Blue') ],
```

Comment:
Could you please use lowercase for red, green, and blue to be consistent with other blocks?